RELEASE_NOTES.md

```diff
@@ -10,7 +10,15 @@
 
 ## New Features
 
-<!-- Here goes the main new features and examples or instructions on how to use them -->
+- There is a new `Receiver.triggered` method that can be used instead of `selected_from`:
+
+  ```python
+  async for selected in select(recv1, recv2):
+      if recv1.triggered(selected):
+          print('Received from recv1:', selected.value)
+      if recv2.triggered(selected):
+          print('Received from recv2:', selected.value)
```

Comment:
```suggestion
          print('Received from recv1:', selected.message)
      if recv2.triggered(selected):
          print('Received from recv2:', selected.message)
```

Reply:
Shouldn't sybil catch this? Ah, probably because it's a RELEASE_NOTES.md file.

Reply:
We are not "testing" the `RELEASE_NOTES.md`. I think we discussed it at some point, my feeling is no, the release notes sometimes have also old code as example of how to migrate some code, that it is not intended to work. I think it would be difficult to test examples in the release notes.

Reply:
fixed.